tests/test-webapps/test-jetty-webapp/pom.xml

```diff
@@ -88,50 +86,55 @@
-          <dependency>
-            <groupId>org.eclipse.jetty</groupId>
-            <artifactId>jetty-client</artifactId>
-            <version>${project.version}</version>
-          </dependency>
-          <dependency>
-            <groupId>org.eclipse.jetty</groupId>
-            <artifactId>jetty-servlets</artifactId>
-            <version>${project.version}</version>
-          </dependency>
-        </dependencies>
-        <configuration>
-          <stopPort>8087</stopPort>
-          <stopKey>foo</stopKey>
-          <scan>1</scan>
-          <systemProperties>
-              <fooprop>222</fooprop>
-          </systemProperties>
-          <webApp>
-            <contextPath>/test</contextPath>
-            <tempDirectory>${project.build.directory}/work</tempDirectory>
-          </webApp>
-          <loginServices>
-            <loginService implementation="org.eclipse.jetty.security.HashLoginService">
-              <name>Test Realm</name>
-              <config>src/main/config/demo-base/etc/realm.properties</config>
-            </loginService>
-          </loginServices>
-        </configuration>
-      </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-deploy-plugin</artifactId>
         <configuration>
-          <!-- must deploy: required for jetty-distribution -->
+          <!-- DEMO Project, Must Deploy -->
           <skip>false</skip>
         </configuration>
       </plugin>
     </plugins>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.eclipse.jetty</groupId>
+          <artifactId>jetty-maven-plugin</artifactId>
+          <version>${project.version}</version>
+          <dependencies>
+            <dependency>
+              <groupId>org.eclipse.jetty</groupId>
+              <artifactId>jetty-client</artifactId>
+              <version>${project.version}</version>
+            </dependency>
+            <dependency>
+              <groupId>org.eclipse.jetty</groupId>
+              <artifactId>jetty-servlets</artifactId>
+              <version>${project.version}</version>
+            </dependency>
+          </dependencies>
+          <configuration>
+            <stopPort>8087</stopPort>
+            <stopKey>foo</stopKey>
+            <scan>1</scan>
+            <systemProperties>
+              <fooprop>222</fooprop>
+            </systemProperties>
+            <webApp>
+              <contextPath>/test</contextPath>
+              <tempDirectory>${project.build.directory}/work</tempDirectory>
+            </webApp>
+            <loginServices>
+              <loginService implementation="org.eclipse.jetty.security.HashLoginService">
+                <name>Test Realm</name>
+                <config>src/main/config/demo-base/etc/realm.properties</config>
+              </loginService>
+            </loginServices>
+          </configuration>
+        </plugin>
+
+      </plugins>
+    </pluginManagement>
```

Comment:
Why has this part been moved from `<build>` to `<pluginManagement>`? We don't need to manage plugins for children POMs, no?

Reply:
Having jetty-maven-plugin as a formal plugin declaration forces it to resolve in the maven reactor.
It was causing a cyclic dependency as  ...

```
        WARNING: not enabled by default, as it introduces a cyclic reference in the build reactor.
        [ERROR] The projects in the reactor contain a cyclic reference:
          Edge between 'Vertex{label='org.eclipse.jetty:jetty-home:10.0.0-SNAPSHOT'}' and
          'Vertex{label='org.eclipse.jetty:test-jetty-webapp:10.0.0-SNAPSHOT'}'
          introduces to cycle in the graph:
           1: org.eclipse.jetty:test-jetty-webapp:10.0.0-SNAPSHOT
           2: org.eclipse.jetty:jetty-maven-plugin:10.0.0-SNAPSHOT
           3: org.eclipse.jetty:jetty-home:10.0.0-SNAPSHOT
           4: org.eclipse.jetty:test-jetty-webapp:10.0.0-SNAPSHOT @
```

Simply moving it to `<pluginManagement>` eliminates it from being involved in the reactor, but still allows it to be used from the command line at will.